src/main/java/racingcar/application/dto/GameRequest.java

```diff
@@ -1,7 +1,10 @@
 package racingcar.application.dto;
 
+import javax.validation.constraints.NotBlank;
+
 public class GameRequest {
 
+    @NotBlank
```

Comment:
어떤 에러상황인지 클라이언트에게 직관적으로 알려주기위해 에러 메시지를 지정해줄 수 있습니다 😄 

```suggestion
    @NotBlank(message = "참가자의 이름은 공백이 될 수 없습니다.")
```